environment.yml

```diff
@@ -18,4 +18,6 @@ dependencies:
  - pytest
  - pytest-runner
  - spyder
+ - pymongo
```

Comment:
pymongo can hopefully be dropped (only used for bson)

Reply:
..waiting for it to be dropped....